utils/pull-upstream.sh

```diff
@@ -152,6 +153,18 @@ case "${FURY_MODULE}" in
     ;;
   "prometheus-adapter")
     populate_package "prometheusAdapter"
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    cd "${WORK_DIR}"
+    helm template \
```

Comment:
Let's pin the version of the chart with the corresponding version of the manifests if possible, as a minimum we should check that what comes out of this command is the same version we are using in the manifests.